mentat/app.py

```diff
@@ -60,15 +76,25 @@ def expand_paths(paths: Iterable[str]) -> Iterable[str]:
     return globbed_paths
 
 
-def run(paths: Iterable[str], exclude_paths: Optional[Iterable[str]] = None):
+def run(
+    paths: Iterable[str],
+    exclude_paths: Optional[Iterable[str]] = None,
+    backup_dir: Optional[str] = ".mentat_backups",
```

Comment:
I don't think this should be 'Optional'; it should always be a string (never None).